best-practices/README.md

```diff
@@ -0,0 +1,71 @@
+Best Practices
+=============
+
+The following are locally agreed upon best practices. As often as possible,
+these will inherit from practices within the larger open source development
+community at large. These best practices may, and likely will, change in the
+future. But that change will be incorporated via group consensus and documented
+here.
+
+* [Documentation](documentation.md)
+* [Project README's](project_readme.md)
+* [Deployment](deployment.md)
+
+
+General
+-------
+
+* These are not to be blindly followed; strive to understand these and ask
+  when in doubt.
+* Don't duplicate the functionality of a built-in library.
+* Don't swallow exceptions or "fail silently."
```

Comment:
Interesting. I can definitely see changing the wording to include the first part of your sentence about logging. That's a good thing to make explicit anyway.

I'm less clear on the use case when we wouldn't want to at least log an exception. Your proposal on documentation seems reasonable assuming we allow that. Do any of those rare cases come to mind?